### PR TITLE
refactor(progress): Add progress role to component for accessibility. #2248

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #

## Summary

refactor(progress): Add progress role to component for accessibility. #2248

### CODE REVIEW

**Comment by driskull** [comment]
Ready for review

File: src/components/calcite-input/calcite-input.tsx [hunk 1]
```diff
@@ -80,6 +80,9 @@ export class CalciteInput {
    * calcite-ui-icon name to this prop to display a requested icon for any input type */
   @Prop({ reflect: true }) icon: string | boolean;
 
+  /** string to override English loading text */
```
Comment (caripizza) [inline src/components/calcite-input/calcite-input.tsx:83]: can you add the JSDoc tag under the description comment for docs-json?
```js
        /**
          * string to override English loading text
          * @default "Loading"
         */
```

File: src/utils/resources.ts [hunk 2]
```diff
@@ -4,3 +4,7 @@ export const CSS_UTILITY = {
   lightTheme: "calcite-theme-light",
   rtl: "calcite--rtl"
 };
+
+export const TEXT = {
+  loading: "Loading"
+};
```
Comment (caripizza) [inline src/utils/resources.ts:10]: 💯

**Review by caripizza — APPROVED** [review]
👍

**Pull request merged**